File: workspace/data-ingestion/src/main/java/de/pfiva/data/ingestion/model/snips/Value.java
package de.pfiva.data.ingestion.model.snips;

public class Value {

	private String kind;
	private String value;
	private String grain;
	private String precision;
	
	public String getKind() {
		return kind;
	}
	public void setKind(String kind) {
		this.kind = kind;
	}
	public String getValue() {
		return value;
	}
	public void setValue(String value) {
		this.value = value;
	}
	public String getGrain() {
		return grain;
	}
	public void setGrain(String grain) {
		this.grain = grain;
	}
	public String getPrecision() {
		return precision;
	}
	public void setPrecision(String precision) {
		this.precision = precision;
	}
	@Override
	public String toString() {
		return "Value [kind=" + kind + ", value=" + value + ", grain=" + grain + ", precision=" + precision + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/snips/Value.ts
export class Value {

    private _kind: string;
    private _value: string;
    private _grain: string;
    private _precision: string;

	public get kind(): string {
		return this._kind;
	}

	public set kind(value: string) {
		this._kind = value;
    }

	public get value(): string {
		return this._value;
	}

	public set value(value: string) {
		this._value = value;
    }
    
	public get grain(): string {
		return this._grain;
	}

	public set grain(value: string) {
		this._grain = value;
	}

	public get precision(): string {
		return this._precision;
	}

	public set precision(value: string) {
		this._precision = value;
	}
}
